content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -847,6 +847,35 @@ not vulnerable to ordering changes in the list.
 
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
+### Force deletion
+
+{{< feature-state feature_gate_name="AllowUnsafeMalformedObjectDeletion" >}}
+
+{{< caution >}}
+This may break the workload associated with the resource being force deleted, if it
+relies on the normal deletion flow, so cluster breaking consequences may apply.
+{{< /caution >}}
+
+By enabling the delete option `ignoreStoreReadErrorWithClusterBreakingPotential`, the
+user can perform an unsafe, and force **delete** operation of an undecryptable/corrupt
```

Comment:
Is there missing a noun after `unsafe`?

We should explain why the operation is considered unsafe.

Reply:
Maybe drop the _and_. I agree about explaining the safety risks more. I presume that this is because it can break invariants the control plane otherwise aims to maintain - have I got that right?

Reply:
Also see my suggestion: https://github.com/kubernetes/website/pull/48463#discussion_r1854276913

Reply:
@stlaz I added `is unsafe because it ignores finalizer constraints, and skips precondition checks.`  and we also have the caution at the top, do you want the doc to mention any other specific concern?